src/oc_erchef/apps/chef_index/src/chef_index_batch.erl

```diff
@@ -144,11 +143,11 @@ flush(State = #chef_idx_batch_state{item_queue = Queue,
     {PidsToReply, Timestamps, DocsToAdd} = lists:unzip3(Queue),
     Doc = wrap(DocsToAdd, State),
     Self = self(),
-    spawn_link(
+    spawn(
```

Comment:
Yes.  We could alternatively add a handle_info for the EXIT messages.  However, I think the best we can do is log a message since we no longer have the list of Pids waiting for a response.  Those  requests will all fail with a gen_server timeout.